tcms/utils/tests/test_github.py

```diff
@@ -8,6 +11,19 @@
 from tcms.utils import github
 
 
+class CalculateSignatureTestCase(test.TestCase):
+    def test_string_arguments(self):
+        with self.assertRaisesRegex(TypeError, 'key: expected bytes or bytearray, but got \'str\''):
+            github.calculate_signature('secret', 'content')
+
+    def test_return_format(self):
+        sha1 = hmac.new(key=b'secret',
+                        msg=b'content',
+                        digestmod=hashlib.sha1).hexdigest()
+        self.assertEqual(github.calculate_signature(b'secret', b'content'),
+                         f'sha1={sha1}')
```

Comment:
The `sha1=` prefix is important and is behavior coming from our own code base, the rest is coming from hashlib and we don't need to check that it's calculations are correct.

The result for a given secret and value is always the same so you can hard-code the expected result inside the assertion instead of calculating it by hand.